src/docs/get-started/codelab-web.md

```diff
@@ -852,7 +852,7 @@ class _SignUpFormState extends State<SignUpForm> {
   @override
   Widget build(BuildContext context) {
     return Form(
-      onChanged: () => _updateFormProgress(),
+      onChanged: () => _updateFormProgress,
```

Comment:
I think the implementation should be:
`onChanged: _updateFormProgress,`
rather than using arrow function.